### DISCUSSION TITLE
How to filter by "Last Sold" Date

### DISCUSSION THREAD

**Discussion by jaamrode** [rgriebl/brickstore#652] [Q&A]
How do I filter based on "Last Sold"?  I am trying to find the inventory that has not sold in the last year so when I try to use the filters I put in "Last Sold" < 1/1/2022 and nothing comes up. No matter what combination of comparisons except "is not" pulls any data.  What am I doing wrong?

**Comment by ZZJHONS** [comment]
You can select a lot and rightclick the column last sold -> create filter with selection, then change it to less equal than. The only thing is missing last sold data will also appear in the list, you then sort by that column and scroll to see the data you want.
